### PR TITLE
VTK Headless Build

### PR DESCRIPTION
**Pull request by HuyNguyenDePaul**
Successfully built VTK. This pull request is to show the compilated VTK build.

### CODE REVIEW

File: db.sqlite3
Comment (alexandru-orhean) [inline db.sqlite3:1]: Remove db.sqlite3

**Review by alexandru-orhean — CHANGES_REQUESTED** [review]
Implement changes

File: db.sqlite3
Comment (HuyNguyenDePaul) [reply db.sqlite3:1]: Fixed this issue.

**Review by alexandru-orhean — APPROVED** [review]
Looks OK to me!

**Pull request merged**